shared/src/business/utilities/getFormattedTrialSessionDetails.setPretrialMemorandumFilter.test.js

```diff
@@ -0,0 +1,174 @@
+import { MOCK_CASE } from '../../test/mockCase';
+import { PARTIES_CODES } from '../entities/EntityConstants';
+const { applicationContext } = require('../test/createTestApplicationContext');
+import { setPretrialMemorandumFiler } from './getFormattedTrialSessionDetails';
+
+describe('formattedTrialSessionDetails', () => {
+  describe('setPretrialMemorandumFiler', () => {
+    const mockPretrialMemorandumDocketEntry = {
+      createdAt: '2018-11-21T20:49:28.192Z',
+      docketEntryId: '9de27a7d-7c6b-434b-803b-7655f82d5e07',
+      docketNumber: '101-18',
+      documentTitle: 'Pretrial Memorandum',
+      documentType: 'Pretrial Memorandum',
+      eventCode: 'PMT',
+      filedBy: 'Test Petitioner',
+      filers: [MOCK_CASE.petitioners[0].contactId],
+      filingDate: '2018-03-01T05:00:00.000Z',
+      index: 5,
+      isFileAttached: true,
+      isOnDocketRecord: true,
+      isStricken: false,
+      partyIrsPractitioner: false,
+      processingStatus: 'complete',
+      receivedAt: '2018-03-01T05:00:00.000Z',
+      userId: '7805d1ab-18d0-43ec-bafb-654e83405416',
+    };
+
+    let mockCase;
+
+    beforeEach(() => {
+      applicationContext
+        .getPersistenceGateway()
+        .getCaseByDocketNumber.mockImplementation(() => mockCase);
+    });
```

Comment:
I'm not sure you're getting much out of this `beforeEach`. The first time it fires before the first test, `mockCase` is still undefined. The second time it fires before the second test, `mockCase` is what the first test had defined, etc.

Plus, I don't think that this Persistence Gateway method is invoked in the function you are testing. Seems safe to remove.

Reply:
removed